benchmark/bootstrap/aws-launch.sh

```diff
@@ -323,7 +322,19 @@ runcmd:
     git clone --depth 100 __CORETH_BRANCH_ARG__ https://github.com/ava-labs/coreth.git
   - >
     sudo -u ubuntu -D /mnt/nvme/ubuntu
-    git clone --depth 100 __LIBEVM_BRANCH_ARG__ https://github.com/ava-labs/libevm.git
+    git clone https://github.com/ava-labs/libevm.git
+  - >
+    sudo -u ubuntu -D /mnt/nvme/ubuntu/libevm
+    git checkout __LIBEVM_COMMIT__
```

Comment:
Why are we checking out main and then switching to this branch? Seems like we can do this in one just like the others.